index.js

```diff
@@ -1,15 +1,25 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
 var Funnel = require('broccoli-funnel');
 
 module.exports = {
   name: 'ember-test-with-data',
-  included(app) {
+  included: function(app) {
     var registry = app.registry;
     var options = app.options || {};
     this.env = app.env;
+
     var configPath = options.configPath;
+
+
+    // NOTE: For ember-cli >= 2.6.0-beta.3, project.configPath() returns absolute path
+    // while older ember-cli versions return path relative to project root
+    if (!path.isAbsolute(configPath)) {
```

Comment:
Because this check happens before the guard for configPath, it creates an error when configPath is undefined.  This is probably solved by dropping it after the guard.